pom.xml

```diff
@@ -67,12 +67,16 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-smallrye-opentracing</artifactId>
+      <artifactId>quarkus-opentelemetry</artifactId>
     </dependency>
     <dependency>
-       <groupId>io.opentracing.contrib</groupId>
-       <artifactId>opentracing-jdbc</artifactId>
-   </dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-extension-trace-propagators</artifactId>
```

Comment:
which propagators so you need from here? The OTel extension will propagate using the default W3C headers.

Reply:
Probably none at the moment. I'll remove it. Thanks.